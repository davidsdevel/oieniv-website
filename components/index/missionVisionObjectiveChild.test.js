/*eslint-env jest*/
import React from "react";
import MissionVisionObjectiveChild from "./missionVisionObjectiveChild";
import {shallow} from "enzyme";
import renderer from "react-test-renderer";

describe("MissionVisionObjectiveChild with Enzyme", () => {
	it("render MissionVisionObjectiveChild without crash", () => {
		const wrapper = shallow(<MissionVisionObjectiveChild title="Mision" content="Hola Mundo" iconURL="https://someURL.com"/>);
	
		expect(wrapper.find("h2").text()).toEqual("Mision");
		expect(wrapper.find("p").text()).toEqual("Hola Mundo");
		expect(wrapper.find("img").prop("src")).toEqual("https://someURL.com");
	});
});

describe("MissionVisionObjectiveChild with Snapshot", () => {
	it("render MissionVisionObjectiveChild without crash", () => {
		const component = renderer.create(<MissionVisionObjectiveChild title="Mision" content="Hola Mundo" iconURL="https://someURL.com"/>);
		const tree = component.toJSON();

		expect(tree).toMatchSnapshot();
	});
});
