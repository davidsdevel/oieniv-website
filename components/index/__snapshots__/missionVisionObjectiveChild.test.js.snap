// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MissionVisionObjectiveChild with Snapshot render MissionVisionObjectiveChild without crash 1`] = `
<div
  className="mission-container"
>
  <div
    className="image-circle"
  >
    <img
      alt="OIENIV Mision"
      src="https://someURL.com"
    />
  </div>
  <h2>
    Mision
  </h2>
  <p>
    Hola Mundo
  </p>
</div>
`;
